ui/main.qml

```diff
@@ -110,21 +116,25 @@ ApplicationWindow {
     
     property bool currentlyHasANotification: false
 
-    onActiveChanged: {
-        if (active && currentlyHasANotification) {
-            currentlyHasANotification = false
-            // QML doesn't have a function to hide notifications, but this does the trick
-            systemTray.hide()
-            systemTray.show()
+    //! Workaround for custom QQuickWindow
+    Connections {
+        target: applicationWindow
+        onActiveChanged: {
```

Comment:
This can be put in the Connections above right? To only use one Connections

Reply:
yes, will fix it.